html/browsers/history/the-location-interface/location-protocol-setter.html

```diff
@@ -78,17 +72,15 @@
     'x†',
     'X\x00',
     'X\x01',
-    'X\x0A',
     'X\x20',
     'X\x21',
     'X\x7F',
     'X\x80',
     'X\xFF',
     'X†',
-    'a\x0A',
-    'a+-.\x0A'
-  ]
-  ;broken.forEach((val) => {
+  ];
+
+  broken.forEach(val => {
     test(() => {
       assert_throws_dom("SyntaxError", () => { location.protocol = val })
     }, encodeURI(val) + " (percent-encoded) is not a scheme")
```

Comment:
Changing this assert to be like the one in -sameish would be good. (Including my suggested change.)